### DISCUSSION TITLE
Dashboard language management

### DISCUSSION THREAD

**Discussion by gnumax** [allinurl/goaccess#2289] [Ideas]
Hi,

Although GoAccess in "Settings" does not allow to change the language, is it foreseen to incorporate the option to change the dashboard language?

Thanks :)

**Comment by allinurl** [comment]
I was looking at this and since it uses `gettext`, so far I haven't found a way to do it a runtime. It would have to somehow include multiple languages at compile-time. Nevertheless I think having this would be awesome.
